webbeans-tomcat/src/main/resources/META-INF/openwebbeans/openwebbeans-shade.properties

```diff
@@ -0,0 +1,160 @@
+#Licensed to the Apache Software Foundation (ASF) under one
+#or more contributor license agreements.  See the NOTICE file
+#distributed with this work for additional information
+#regarding copyright ownership.  The ASF licenses this file
+#to you under the Apache License, Version 2.0 (the
+#"License"); you may not use this file except in compliance
+#with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing,
+#software distributed under the License is distributed on an
+#"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#KIND, either express or implied.  See the License for the
+#specific language governing permissions and limitations
+#under the License.
+
+#This is for reference if the shade auto appending is too messed up
+
+org.apache.webbeans.spi.SecurityService=org.apache.webbeans.web.tomcat.TomcatSecurityService
```

Comment:
everything which is "default" is not needed and it misses an ordinal to override less prioritized SPI files, we have an openwebbeans-maven module with a shade transformer implementation for that purpose, sounds like it would be less error prone, wdyt?